sdk/send-react/src/components/SendCowri/index.jsx

```diff
@@ -0,0 +1,150 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { ThemeProvider } from '@material-ui/styles'
+
+import BigNumber from 'bignumber.js'
+
+import isPromise from 'is-promise'
+
+import { isAddress } from 'web3-utils'
+import ShellManager from '@cowri/cowri-dapp-sdk'
+
+import theme from './theme'
+import getWeb3 from './getWeb3'
+
+import CowriSendButton from './CowriSendButton'
+import CowriSendModal from './CowriSendModal'
+import CowriSpinner from './CowriSpinner'
+
+function SendCowri (props) {
+  const {
+    address,
+    amount,
+    web3
+  } = props 
+
+  const [ shellManager, setShellManager ] = React.useState(null)
+
+  const [ sendState, setSendState ] = React.useState({
+    address: address || '',
+    addressError: false,
+    amount: amount || '',
+    amountError: false,
+    fixedAddress: !!address,
+    fixedAmount: !!amount,
+    sending: false
+  })
+
+  const [ spinnerState, setSpinnerState ] = React.useState({
+    error: false,
+    message: '',
+    spinning: false,
+    success: false
+  })
+
+  React.useEffect(() => {
+    if (web3) getWeb3().then(setSM)
+    else if (isPromise(web3)) web3().then(setSM)
+    else setSM(web3)
+
+    function setSM (web3) { setShellManagerm(new ShellManager(web3)) }
```

Comment:
extra m in setShellManager
```suggestion
    function setSM (web3) { setShellManager(new ShellManager(web3)) }
```